src/main/resources/mapper/PointMapper.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.example.dao.PointDAO">
+    <insert id="insertPoint" keyColumn="id" keyProperty="id" useGeneratedKeys="true">
+        INSERT INTO points (id, xCoordinate, yCoordinate) VALUES (#{id}, #{xCoordinate}, #{yCoordinate})
+    </insert>
+    <delete id="deletePoint">
+        DELETE FROM points WHERE id = #{id}
+    </delete>
+    <resultMap id="pointsResult" type="org.example.model.Point">
+        <result property="id" column="id"/>
+        <result property="xCoordinate" column="xCoordinate"/>
```

Comment:
same here - <result property="xCoordinate" column="x_coordinate"/> and so on

Reply:
Got it